sdk/communication/communication-common/README.md

```diff
@@ -73,7 +73,7 @@ const userCredential = new AzureCommunicationUserCredential({
 
```

Comment:
We have more READMEs where we had to comment links, e.g. the ones under samples (for example https://docs.microsoft.com/javascript/api/@azure/communication-administration)

There we commented links to the API references and to individual code files.